crates/turbopack-node/src/render/mod.rs

```diff
@@ -51,8 +58,61 @@ enum RenderStaticIncomingMessage {
         headers: Vec<(String, String)>,
         body: String,
     },
+    Headers {
+        data: ResponseHeaders,
+    },
     Rewrite {
         path: String,
     },
     Error(StructuredError),
 }
+
+#[derive(Deserialize)]
+#[serde(tag = "type", rename_all = "camelCase")]
+enum RenderBodyChunks {
+    BodyChunk { data: Vec<u8> },
+    BodyEnd,
+    Error(StructuredError),
+}
+
+pub(crate) fn stream_body_chunks(
+    mut operation: NodeJsOperation,
+    intermediate_asset: AssetVc,
+    intermediate_output_path: FileSystemPathVc,
+    project_dir: FileSystemPathVc,
+) -> Body {
+    let chunks = Stream::new_open(
+        vec![],
+        Box::pin(generator! {
```

Comment:
This is a bit more difficult because of the 2 paths that have different return types. I was attempting to reuse code, and because the stream doesn't need to read any Vcs, I thought it would be ok (the node eval streaming did need to read Vcs).

For proxy results which only have a streamed result, it'd be ok to use the same manual cell approach. But for static rendering, I would need to completely change the return type and wouldn't be able to share code.

Reply:
It read VCs, e.g. recv() applies source maps which reads VCs, also structured error print uses Vcs